huxley/settings/logging.py

```diff
@@ -40,7 +40,7 @@
         'huxley.api': {
             'handlers': ['database'],
             'level': 'DEBUG',
-            'propagate': True, #Set to false to elimninate logs in the console
+            'propagate': False, #Set to false to elimninate logs in the console
```

Comment:
It is just to keep the login suppressed when running the tests. We need to change the way that we are logging this data at some point.